### PR TITLE
feat: Create base page component

### PR DESCRIPTION
**Pull request by darryltec**
## Changes

- Updated markdown to support basic usage

### Added

- Created a page component that would extend to have more than a heading and intro

image

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/Markdown/Markdown.test.tsx [hunk 1]
```diff
@@ -1,168 +1,71 @@
...
-      className="base"
-    >
-      <p
-        className="base regular base greyBlueDark"
-      >
-        Paragraph
-      </p>
-    </div>
-  `);
+  expect(tree).toMatchSnapshot();
```
Comment (seanhealy) [inline packages/components/src/Markdown/Markdown.test.tsx]: I wonder if we should update the `plop` generator to not use inline snapshot…

File: packages/components/src/Markdown/Markdown.tsx [hunk 2]
```diff
@@ -26,22 +32,37 @@ interface BaseRendererProps {
   children: ReactNode;
 }
 
-export function Markdown({ content, externalLink }: MarkdownProps) {
+export function Markdown({ content, externalLink, basicUsage }: MarkdownProps) {
+  const props = {
+    ...(basicUsage && {
+      disallowedTypes: ["paragraph", "heading"],
```
Comment (seanhealy) [inline packages/components/src/Markdown/Markdown.tsx]: Should we allow `images` and `tables`, etc here?

https://www.markdownguide.org/cheat-sheet

File: packages/components/src/Markdown/Markdown.test.tsx (hunk 1 above)
Comment (darryltec) [reply packages/components/src/Markdown/Markdown.test.tsx]: Yah, I plan to update the default content too.. I'll add a task

File: packages/components/src/Markdown/Markdown.tsx (hunk 2 above)
Comment (darryltec) [reply packages/components/src/Markdown/Markdown.tsx]: nope!

**Pull request merged**